segment.py

```diff
@@ -48,17 +50,19 @@ def lambda_handler(event, context):
             # )
 
             # Call ffmpy function
-            segment()
+            segment(LocalPath)
             global destination
 
             # Each chunk is uploaded to s3
+            FilePath, Extension = os.path.splitext(LocalPath)
             print "Uploading segments to s3..."
-            for filename in os.listdir('/tmp/SEGMENT*'):
-                destination = 'Segmented/{}'.format(filename)
-                s3_client.upload_file('/tmp/'+filename, bucket, destination)
+            for filename in os.listdir('/tmp/{}/{}*'.format(ConversionID, FilePath)):
+                if !filename.endswith('.mp3'):
```

Comment:
I think we can put mp3 in same directory as everything else, and remove this logic